napari/layers/image/image.py

```diff
@@ -189,6 +196,7 @@ def __init__(
         blending='translucent',
         visible=True,
         multiscale=None,
+        plane_thickness=10,
```

Comment:
Is there any reason we've put just `plane_thickness` in init instead of all `plane_` options? Maybe for now leave it out and we can put them in later?